### ISSUE TITLE
Data Labels Color

### ISSUE THREAD

**Issue by jkinley** [naver/billboard.js#589]
## Description

Is it possible to change the text color on data-labels? If yes, can we have some clearer documentation on how to do so?
## Steps to check or reproduce

I am trying to place data labels on a bar chart and the text colors are randomly assigned.

**Comment by netil** [comment]
Hi @jkinley-sds, basically data label text colors are based on the dataset's color.
If you want to change a specific one, you can do by manually.

For example, if you want to update the 3rd data of 'data1' to be blue, you can add following option:

```js
var chart = bb.generate({
  data: {
    type: 'bar',
    columns: [
      ["data1", 30, 200, 100, 400, 150, 250],
      ["data2", 130, 100, 140, 200, 150, 50]
    ],
    labels: true
  },
  onrendered: function() {
    // change 3rd data label text color of 'data1'
    this.main.select(".bb-texts-data1 .bb-text-2")
      .style("fill", "blue");
  }
});
```
- working demo: https://jsbin.com/notukic/edit?html,js,output

**Comment by elyobo** [comment]
Seems like something worth making a standard option; with the use of the `data.labels.position` `x` and `y` options, it's possible (and, in our case, desirable) to put the labels on top of the bars which I imagine is a not uncommon requirement.  Example to illustrate below

![image](https://user-images.githubusercontent.com/407806/50134845-23ed5200-02e6-11e9-93ec-98cc54e1c874.png)

**Comment by netil** [comment]
Implemented by #871
